apps/passport-client/src/dispatch.ts

```diff
@@ -598,42 +590,34 @@ async function sync(state: AppState, update: ZuUpdate) {
       downloadingPCDs: true
     });
 
-    /**
-     * Get both PCDs and subscriptions
-     * Subscriptions might be null even if we got PCDs, if we were
-     * downloading from a pre-v3 version of encrypted storage
-     * {@link SyncedEncryptedStorageV3}
-     * */
-    const downloaded = await downloadStorage();
-
-    if (downloaded != null && downloaded.pcds != null) {
-      const { pcds, subscriptions } = downloaded;
-
-      if (subscriptions) {
-        addDefaultSubscriptions(subscriptions);
```

Comment:
I don't think this should be removed (or, if it is removed, it should be replaced by an equivalent call elsewhere).

`addDefaultSubscriptions` is the means by which we opt users into new feeds, and I could imagine us wanting to update everyone's subscriptions to add a "Frogcrypto" feed. The above call ensures that the user's subscriptions include all of the default subscriptions immediately after loading it from storage.

Reply:
It wasn't removed.  It was moved earlier and no longer conditional, since subscriptions is never null.  See line 599 on the right.